pp/integrations/plugin-packs/procedures/operatingsystems-linux-snmpv3.md

```diff
@@ -0,0 +1,965 @@
+| --statefile-format                              | Define the format used to store the cache. Available formats: 'dumper', 'storable', 'json' (default).                                                                                                                                         |
+| --statefile-key                                 | Define the key to encrypt/decrypt the cache.                                                                                                                                                                                                  |
+| --statefile-cipher                              | Define the cipher algorithm to encrypt the cache (default: 'AES').                                                                                                                                                                            |
+| --warning-usage                                 | Warning threshold.                                                                                                                                                                                                                            |
+| --critical-usage                                | Critical threshold.                                                                                                                                                                                                                           |
+| --warning-access                                | Warning threshold.                                                                                                                                                                                                                            |
+| --critical-access                               | Critical threshold. Check if storage is readOnly: --critical-access=readOnly                                                                                                                                                                  |
+| --add-access                                    | Check storage access (readOnly, readWrite).                                                                                                                                                                                                   |
+| --units                                         | Units of thresholds (default: '%') ('%', 'B').                                                                                                                                                                                                |
+| --free                                          | Thresholds are on free space left.                                                                                                                                                                                                            |
```

Comment:
```suggestion
| --free                                          | Apply thresholds on free space instead of used space.                                                                                                                                                                                                            |
```